src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -53,34 +55,62 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     resource_group_name=None,
                     target=None,
                     auth_mode=None):
+
+    registry, resource_group_name = validate_managed_registry(
```

Comment:
super nit: we can move this back to where it was as it's not used before.

Reply:
As per discussion - we should be removing this validation all together.

Reply:
Dropped the registry validation for task group.